USER-INTERFACE-INTERFACE.md

```diff
@@ -386,6 +405,66 @@ database password. If you want to know whether the password you have is the corr
   try to connect to when it starts up next time.  It's a secret, so if you don't supply the `dbPasswordOpt` in the
   request you won't see it.
 
+* `paymentCurves`: These are parameters that define the Node's regard to quality of financial records created during interaction with
+  other Nodes. The goal is to determine whether the records have crossed boarders meaning that a certain action needs to be taken 
+  - either to make a payment to the touched Nodes in order to redeem our debts or put those Nodes under ban since they've too long 
+  ignored their liabilities toward us. These are ever present values, no matter if the user's set any value, they have defaults.
+
+* `balanceDecreasesForSec`: This interval begins after payment_suggested_after_sec for payables and after 
+  payment_suggested_after_sec + payment_grace_before_ban for receivables. During the interval, the amount of a payable that is allowed
+  to remain unpaid, or a receivable that doesn't cause a ban, decreases linearly from the balance_to_decrease_from to
+  permanent_debt_allowed_gwei or unban-when_balance_below_gwei.
```

Comment:
Is `unban-when_balance_below_gwei` supposed to be `unban_when_balance_below_qwei`